src/ontology/components/mappings.owl

```diff
@@ -4554,5 +4552,5 @@
 
 
 
-<!-- Generated by the OWL API (version 4.5.29) https://github.com/owlcs/owlapi -->
+<!-- Generated by the OWL API (version 4.5.26.2023-07-17T20:34:13Z) https://github.com/owlcs/owlapi -->
```

Comment:
Has this file been modified with Protégé?

This file is automatically generated from the mappings between CL and foreign ontologies. It should not be manually edited.